### PR TITLE
feat(assets): add op22 row table

### PR DESCRIPTION
**Pull request by masthom**

### CODE REVIEW

File: src/assets/data/edition/series/1/section/2/op22/source-description.json [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+                ],
+                "writingMaterial": "Notenpapier, 8 Systeme (oben und unten beschnitten), Format (quer): 170–172 × 270 mm, kein Firmenzeichen (Bl. 1);<br />Notenpapier, 8 Systeme (oben und unten beschnitten), Format (quer): 170–172 × 270 mm, kein Firmenzeichen (Bl. 1);<br />Notenpapier, 9 Systeme (oben, unten und rechts beschnitten), Format (quer): 135 × 255–260 mm, kein Firmenzeichen (Bl. 2);",
+                "writingInstruments": {
+                    "main": "Bleistift",
+                    "secondary": [
+                        "blauer Buntstift", "grüner Buntstift", "roter Buntstift", "schwarze Tinte"
+                    ]
+                },
+                "title": "<em>Reihen: Quartett, op</em>[.]<em> 22 </em>[blauer Buntstift] <em>| Geige, Klar., Saxophon | u. Klavier| 1929/1930 </em>[Bleistift] auf Bl. 1<sup>r</sup> oben rechts; <br /><em>Quart. op. 22 </em>auf Bl. 2<sup>r</sup> oben rechts mit blauem Buntstift; <br /><em>Quart. | op</em>[.]<em> 22</em> auf Bl. 3<sup>r</sup> oben rechts mit blauem Buntstift.",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/source-description.json:20]:
```suggestion
                "title": "<em>Reihen: Quartett, op</em>[.] <em>22</em> [blauer Buntstift] <em>| Geige, Klar., Saxophon | u. Klavier| 1929/1930</em> [Bleistift] auf Bl. 1<sup>r</sup> oben rechts; <br /><em>Quart. op. 22</em> auf Bl. 2<sup>r</sup> oben rechts mit blauem Buntstift; <br /><em>Quart. | op</em>[.] <em>22</em> auf Bl. 3<sup>r</sup> oben rechts mit blauem Buntstift.",
```

File: src/assets/data/edition/series/1/section/2/op22/source-description.json [hunk 2]
```diff
@@ -0,0 +1,606 @@
...
+                        "item": "M 305 Sk# / M 308 Sk# / M 309 Sk#",
+                        "itemLinkTo": "SkRT",
+                        "itemDescription": "(Reihentabelle op. 22)",
+                        "folios": [
+                            {
+                                "folio": "1r",
+                                "folioLinkTo": "",
+                                "folioDescription": "",
+                                "systemGroups": [
+                                    [],
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/source-description.json:37]:
```suggestion
```

Comment (musicEnfanthen) [reply src/assets/data/edition/series/1/section/2/op22/source-description.json:37]: Remove empty array?

File: src/assets/data/edition/series/1/section/2/op22/source-description.json [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
+                                        }
+                                    ]
+                                ]
+                            },
+                            {
+                                "folio": "2r",
+                                "folioLinkTo": "",
+                                "folioDescription": "",
+                                "systemGroups": [
+                                    [],
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/source-description.json:221]:
```suggestion
```

File: src/assets/data/edition/series/1/section/2/op22/source-description.json [hunk 4]
```diff
@@ -0,0 +1,606 @@
...
+                                        }
+                                    ]
+                                ]
+                            },
+                            {
+                                "folio": "3r",
+                                "folioLinkTo": "",
+                                "folioDescription": "",
+                                "systemGroups": [
+                                    [],
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/source-description.json:405]:
```suggestion
```

File: src/assets/data/edition/series/1/section/2/op22/folio-convolute.json [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                                    "startSystem": 1,
+                                    "endSystem": 8
+                                }
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "folioId": "3v",
+                    "systems": "7",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/folio-convolute.json:96]: 7 correct?

File: src/assets/data/edition/series/1/section/2/op22/folio-convolute.json [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+                {
+                    "folioId": "3r",
+                    "systems": "8",
+                    "format": {
+                        "height": 170,
+                        "width": 270
+                    },
+                    "content": [
+                        {
+                            "id": "SkRTb",
```
Comment (musicEnfanthen) [inline src/assets/data/edition/series/1/section/2/op22/folio-convolute.json:80]:
```suggestion
                            "id": "SkRTc",
```

**Review by musicEnfanthen — COMMENTED** [review]
Many thanks, @masthom . Added a few comments.

File: src/assets/data/edition/series/1/section/2/op22/folio-convolute.json (hunk 5 above)
Comment (masthom) [reply src/assets/data/edition/series/1/section/2/op22/folio-convolute.json]: ofc not! see commit e50272bf9b99eb500625c843c544b273ced8ee94.

**Pull request merged**